src/Calendar.js

```diff
@@ -31,8 +32,9 @@ export default class Calendar extends Component {
 
   handleDateSelected(evt) {
     const dateBefore = !isBefore(evt.target.dataset.date, new Date(format(this.props.minDate, 'MM/DD/YYYY')))
+    const dateAfter = !isAfter(evt.target.dataset.date, new Date(format(this.props.maxDate, 'MM/DD/YYYY')))
 
-    if (dateBefore) {
+    if (dateBefore && dateAfter) {
```

Comment:
this is supposed to be like 'not before this date and not after' right? 

if so could we update verbiage?

Reply:
Yeah, agreed. I'll update the verbiage.